xls/delay_model/op_module_generator.py

```diff
@@ -76,13 +78,32 @@ def generate_ir_package(op: str,
       ("bits[32]", "bits[16]").
     attributes: Attributes to include in the operation mnemonic. For example,
       "new_bit_count" in extend operations.
+    operand_attributes: Attributes consisting of an operand to include in the
+      operation mnemonic. Each element of the sequence specifies the attribute
+      name and the index of the first operand (within 'operand_types').
+      For example, used for "defualt" in sel operations.
```

Comment:
default

Reply:
It seems that you and I took different approaches to solve the same problem at the same time.  Removed my solution since yours is already in the repo.